topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -0,0 +1,290 @@
+
+#### Dependency Resolution Management
+
+{id="configuration.dependencyResolutionManagement"}
+
+To access the IntelliJ Platform Gradle Plugin within the <path>settings.gradle.kts</path> to use with `dependencyResolutionManagement`, add:
+
+```kotlin
+import org.jetbrains.intellij.platform.gradle.extensions.intellijPlatform
+
+plugins {
+    id("org.jetbrains.intellij.platform.settings") version "%intellij-platform-gradle-plugin-version%"
+}
+
+dependencyResolutionManagement {
+    repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
+
+    repositories {
+        mavenCentral()
+    
+        intellijPlatform {
+            recommended()
+        }
+    }
+}
+```
+
+#### Cache Redirector
+
+{id="configuration.cacheRedirector"}
+
+Some repositories, by default, point to JetBrains Cache Redirector to provide better resources resolution.
+However, it is possible to use the direct repository URL, if available.
+
+To switch off the default usage of JetBrains Cache Redirector, add to <path>gradle.properties</path>:
+
+```
+org.jetbrains.intellij.platform.buildFeature.useCacheRedirector=false
+```
+
+### Setting up IntelliJ Platform
+
+Dependencies and [repositories](#configuration.repositories) are handled using explicit entries within `dependencies {}` and `repositories {}` blocks in <path>build.gradle.kts</path> file.
+
+A minimum configuration for targeting IntelliJ IDEA Community 2023.3:
+
+```kotlin
+repositories {
+  mavenCentral()
+
+  intellijPlatform {
+    recommended()
+  }
+}
+
+dependencies {
+  intellijPlatform {
+    intellijIdeaCommunity("2023.3")
+  }
+}
+```
+
+The `intellijIdeaCommunity` in the previous sample is one of extension functions available for adding IntelliJ Platform dependencies to the project.
+Other IDEs can be targeted using one the extensions listed in the table below.
+
+| Extension               | Target IDE                                 |
+|-------------------------|--------------------------------------------|
+| `androidStudio`         | [Android Studio](android_studio.md)        |
+| `clion`                 | [CLion](clion.md)                          |
+| `fleetBackend`          | Fleet Backend                              |
+| `gateway`               | Gateway                                    |
+| `goland`                | [GoLand](goland.md)                        |
+| `intellijIdeaCommunity` | [IntelliJ IDEA Community Edition](idea.md) |
+| `intellijIdeaUltimate`  | [](idea_ultimate.md)                       |
+| `phpstorm`              | [PhpStorm](phpstorm.md)                    |
+| `pycharmCommunity`      | [PyCharm Community Edition](pycharm.md)    |
+| `pycharmProfessional`   | [PyCharm Professional](pycharm.md)         |
+| `rider`                 | [Rider](rider.md)                          |
+| `rustRover`             | Rust Rover                                 |
+| `writerside`            | Writerside                                 |
+
+#### Parametrized IntelliJ Platform dependency
```

Comment:
"Parametrize IntelliJ Platform Dependency" (remove "d" from parametrized and capitalize dependeny)